vcell-rest/src/test/java/org/vcell/restq/apiclient/UsersApiTest.java

```diff
@@ -164,6 +170,11 @@ public void testOldAPITokenGeneration() throws ApiException {
         Assertions.assertEquals("Administrator", token.getUserId());
     }
 
+    /**
+     * If there is no user mapping for the client or the user does not have an JWT token in the Authorization header for HTTP,
+     * throw 401. If the user is a guest, return a token with the user id "vcellguest" and the user key "140220477".
```

Comment:
"if the user is a guest" is misleading to me because it implies that with respect to the UsersApi, the user identified itself as a "guest", but rather the usersResourceApi.getGuestLegacyToken() should not require authentication at all.